modules/rest-util/test/blaze/middleware/fhir/resource_test.clj

```diff
@@ -62,7 +62,7 @@
       [:body :issue 0 :code] := #fhir/code"invalid"
       [:body :issue 0 :diagnostics] := "No content to map due to end-of-input\n at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1]"))
 
-  (testing "body with invalid JSON"
+  (testing "body with invalid payload (JSON)"
```

Comment:
"invalid JSON" is ok here, because "x" is especially a non-valid JSON content, where the following 1 is valid JSON, but not an object